dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -1088,6 +1131,33 @@ public void deleteWorkflowTask(final WorkflowTask task, final User user)
 				+ " has been deleted by the user: " + user.getUserId());
 	}
 
+	@Override
+	@WrapInTransaction
+	public void deleteWorkflowTaskByWebAsset(final String webAsset, final User user) throws DotDataException {

```

Comment:
Should be `deleteWorkflowTaskByContentletId`.  Also, do we need to respect the langauge Id?  Maybe this should take the contentlet object that would include the language Id.

Reply:
done